README.md

```diff
@@ -69,6 +69,70 @@ make CMAKE_FLAGS="-DAPPS_SELECTION=${NAME_OF_APP}"
 [draw]: sample_apps/draw
 [perfbench]: sample_apps/perfbench
 
+
+#### Execute Wasm Application in Place
```

Comment:
having this section here gives an impression that this is the primary way to run an edge app.
is it the case these days?